src/controllers/secure/AdminView.ts

```diff
@@ -2,15 +2,33 @@ import { Get, Hidden, View } from "@tsed/schema";
 import { Controller } from "@tsed/di";
 import { Authorize } from "@tsed/passport";
 import type { CustomUserInfoModel } from "../../model/auth/CustomUserInfoModel.js";
-import { Req } from "@tsed/common";
+import { Req, Res } from "@tsed/common";
 
 @Controller("/")
 @Hidden()
 export class AdminView {
     @Get()
     @Authorize("loginAuthProvider")
-    @View("/secure/admin.ejs")
-    public showAdmin(@Req() req: Req): unknown {
+    public showAdmin(@Req() req: Req, @Res() res: Res): unknown {
+        res.redirect("/admin/files");
+        return {
+            user: req.user as CustomUserInfoModel,
+        };
+    }
+
+    @Get("/files")
+    @Authorize("loginAuthProvider")
+    @View("/secure/files.ejs")
+    public showFileAdmin(@Req() req: Req): unknown {
+        return {
+            user: req.user as CustomUserInfoModel,
+        };
+    }
+
+    @Get("/stats")
+    @Authorize("loginAuthProvider")
+    @View("/secure/stats.ejs")
+    public showStatistics(@Req() req: Req): unknown {
         return {
             user: req.user as CustomUserInfoModel,
```

Comment:
does stats need the user? don't think it does, although can correct me if i am wrong

Reply:
Removed